apps/catalogue/src/views/VariableExplorer.vue

```diff
@@ -84,8 +87,9 @@
                   </router-link>
                 </li>
               </ul>
-              <template v-if="$route.query.tab === 'harmonization'">
-                <harmonization-view />
+              <Spinner v-if="isLoading" />
```

Comment:
please do put spinners everywhere , this creates a janky ui due to multiple redraws

Reply:
please advice? what do you mean?

Reply:
If I would redesign I would centralize the loading of the whole page so we would have only one loading experience (ideally not using spinners but instead by having greyed out boxes or something so the layout doesn't jump).

Reply:
I removed most isLoading spinners